shell/platform/windows/client_wrapper/BUILD.gn

```diff
@@ -97,3 +98,59 @@ executable("client_wrapper_windows_unittests") {
     "//third_party/dart/runtime:libdart_jit",
   ]
 }
+
+if (host_os == "win") {
+  client_wrapper_file_archive_list = [
+    "core_implementations.cc",
+    "engine_method_result.cc",
+    "plugin_registrar.cc",
+    "README",
+    "standard_codec.cc",
+  ]
+
+  win_client_wrapper_file_archive_list = [
+    "flutter_engine.cc",
+    "flutter_view_controller.cc",
+  ]
+
+  zip_bundle("client_wrapper_archive") {
+    output = "$full_platform_name/flutter-cpp-client-wrapper.zip"
+    deps = [
+      ":client_wrapper_windows",
+      "//flutter/shell/platform/common/client_wrapper:client_wrapper",
+      "//flutter/shell/platform/windows/client_wrapper:publish_wrapper_windows",
```

Comment:
This target is in this file.

Reply:
Nice catch. Rewrote it as relative import.